shell/src/main/java/org/apache/accumulo/shell/commands/ScanCommand.java

```diff
@@ -285,6 +296,33 @@ protected void fetchColumns(final CommandLine cl, final ScannerBase scanner,
     }
   }
 
+  private void fetchColumsWithCFAndCQ(CommandLine cl, Scanner scanner, ScanInterpreter interpeter) {
+    String cf = "";
+    String cq = "";
+    if (cl.hasOption(scanOptCf.getOpt())) {
+      cf = cl.getOptionValue(scanOptCf.getOpt());
+    }
+    if (cl.hasOption(scanOptCq.getOpt())) {
+      cq = cl.getOptionValue(scanOptCq.getOpt());
+    }
+
+    if (cf.isEmpty() && !cq.isEmpty()) {
+      String formattedString =
+          String.format("Option - %s when used with (%s" + " cannot be empty )", scanOptCf.getOpt(),
+              scanOptCq.getOpt(), scanOptCq.getOpt());
```

Comment:
```suggestion
          String.format("Option -%s is required when using -%s.", scanOptCf.getOpt(),
              scanOptCq.getOpt());
```
Some suggestions are wording to make it easier to understand. Also, extra parameter was removed and fixed spacing.